docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md

```diff
@@ -1,90 +1,86 @@
+> [!Important]
+> The package identifier must be unique across nuget.org or other host. Publishing makes the package publicly visible, so if you use the example AppLogger library or other test library, use a unique name that includes `Sample` or `Test`.
 
-1. Add any optional properties described on [NuGet metadata properties](/dotnet/core/tools/csproj#nuget-metadata-properties).
+You can add any optional properties described in [NuGet metadata properties](/dotnet/core/tools/csproj#nuget-metadata-properties).
 
-    > [!Note]
-    > For packages built for public consumption, pay special attention to the **PackageTags** property, as tags help others find your package and understand what it does.
+> [!Note]
+> For packages you build for public consumption, pay special attention to the `PackageTags` property. Tags help others find your package and understand what it does.
 
 ## Run the pack command
 
-To build a NuGet package (a `.nupkg` file) from the project, run the `dotnet pack` command, which also builds the project automatically:
+To build a NuGet package or *.nupkg* file from the project, run the [dotnet pack](/dotnet/core/tools/dotnet-pack) command, which also builds the project automatically.
 
 ```dotnetcli
-# Uses the project file in the current folder by default
 dotnet pack
 ```
 
-The output shows the path to the `.nupkg` file:
+The output shows the path to the *.nupkg* file:
 
 ```output
-Microsoft (R) Build Engine version 15.5.180.51428 for .NET Core
-Copyright (C) Microsoft Corporation. All rights reserved.
-
-  Restore completed in 29.91 ms for D:\proj\AppLoggerNet\AppLogger\AppLogger.csproj.
-  AppLogger -> D:\proj\AppLoggerNet\AppLogger\bin\Debug\netstandard2.0\AppLogger.dll
-  Successfully created package 'D:\proj\AppLoggerNet\AppLogger\bin\Debug\AppLogger.1.0.0.nupkg'.
-```
+MSBuild version 17.3.0+92e077650 for .NET
+  Determining projects to restore...
+  Restored C:\Users\myname\source\repos\AppLogger\AppLogger.csproj (in 64 ms).
+  AppLogger -> C:\Users\myname\source\repos\AppLogger\bin\Debug\net6.0\AppLogger.dll
+  Successfully created package 'C:\Users\myname\source\repos\AppLogger\bin\Debug\Contoso.08.28.22.001.Test.1.0.0.nupkg'.
+  ```
 
 ### Automatically generate package on build
 
-To automatically run `dotnet pack` when you run `dotnet build`, add the following line to your project file within `<PropertyGroup>`:
+To automatically run `dotnet pack` whenever you run `dotnet build`, add the following line to your project file within `<PropertyGroup>`:
 
 ```xml
-<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 ```
 
 ## Publish the package
 
-Once you have a `.nupkg` file, you publish it to nuget.org using the `dotnet nuget push` command along with an API key acquired from nuget.org.
+Publish your *.nupkg* file to nuget.org by using the [dotnet nuget push](/dotnet/core/tools/dotnet-nuget-push) command with an API key you get from nuget.org.
```

Comment:
the nuget.org FAQ has an entry about test packages: https://docs.microsoft.com/en-us/nuget/nuget-org/nuget-org-faq#is-it-recommended-to-upload-my-test-packages-to-nuget-org

it suggests using int.nugettest.org, rather than nuget.org.  Perhaps the quickstart should follow the same recommendation?

Reply:
Added a note.